codes/quantum/qubits/quantum_reed_muller.yml

```diff
@@ -5,26 +5,45 @@
 
 code_id: quantum_reed_muller
 physical: qubits
-logical: qubits
+logical: qubits # also qudits
+
+name: 'Quantum Reed-Muller'
 
-name: 'Quantum Reed-Muller code'
 introduced: '\cite{arXiv:quant-ph/9608026}'
 
-description: 'Stub.'
+description: 'Quantum Reed-Muller codes are formed from their classical Reed-Muller counterparts in which polynomials over finite fields encode data. This is done by transforming these polynomials into the stabilizer generator matrices'
+
+protection: 'Protects against Pauli noise.'
+
+features:
+
+  rate: '\(\frac{k}{n}\), where \(k = 2^r - {r \choose t} + 2 \sum_{i=0}^{t-1} {r \choose i}\).'
+
+  encoders: 
+    - 'Standard efficient CSS encoders \cite{arXiv:9608026}'
 
-protection: ''
+  general_gates:
+    - 'Magic state distillation in all prime dimensions \cite{doi:10.1103/PhysRevX.2.041021}'
+    
+  decoders:
+    - 'Standard efficient CSS encoders \cite{arXiv:9608026}'
 
-#notes:
-#  - ''
+  fault_tolerance:
+    - 'Gate switching protocol for universal computation \cite{arxiv:1403.2734}.'
 
-#realizations:
-#  - ''
+  threshold: 
+    - 'Between \(10^{-3}\) and \(10^{-6}\), see \cite{doi:10.1088/2058-9565/abb027}'
+  
+notes:
```

Comment:
Move first item to detail field of a cousin entry with code_id: quantum_convolutional
Move second item to detail field of a cousin entry with code_id: triorthogonal
Move third item to rate field